api/app.js

```diff
@@ -4,7 +4,7 @@ const morgan = require('morgan');
 const path = require('path');
 const db = require('./models');
 const app = express();
-const PORT = process.env.PORT || 8000;
+const PORT = process.env.PORT || 8080;
```

Comment:
I changed it to 8080 because I had something running on 8000. I changed PORT to 8080 in .env but it wasn't being changed, so I changed the port in app.js.